lib/clif-backend/src/code.rs

```diff
@@ -1091,7 +1108,12 @@ impl FunctionCodeGenerator<CodegenError> for CraneliftFunctionCodeGenerator {
         Ok(())
     }
 
-    fn feed_event(&mut self, event: Event, _module_info: &ModuleInfo) -> Result<(), CodegenError> {
+    fn feed_event(
+        &mut self,
+        event: Event,
+        _module_info: &ModuleInfo,
+        loc: u32,
```

Comment:
Can we rename this to `source_loc`, I know it's more verbose but it's also clearer:

```suggestion
        source_loc: u32,
```